src/locales/tr.json

```
@@ -1,7 +1,13 @@
 {
   "welcome": {
     "title": "{programName}`a Hoşgeldiniz",
-    "noTabs": "Şu anda sekme yok. Sağ taraftaki {addTab} öğesini tıklayarak bir tane ekleyin"
+    "noTabs": "Şu anda sekme yok. Sağ taraftaki {addTab} öğesini tıklayarak bir tane ekleyin",
+    "linuxDetected": "Linux'ta olduğunuzu tespit ettik.",
+    "linuxGuide1": "Sesleri çalmak için sağ üstteki çıktı uygulamasını seçmeniz yeterlidir.",
+    "linuxGuide2": "Giriş veya çıkış cihazınızı hiçbir yerde değiştirmeniz gerekmez.",
+    "windowsDetected": "Windows'ta olduğunuzu tespit ettik.",
+    "windowsGuide1": "{SeenOnWebsite} kurulum adımlarını uyguladığınızdan emin olun",
```

This has to be `seenOnWebsite`. It's a key for the line below (`windowsGuide2` that will be inserted there).
```suggestion
    "windowsGuide1": "{seenOnWebsite} kurulum adımlarını uyguladığınızdan emin olun",
```